docs/pages/Builder.md

```diff
@@ -83,6 +84,55 @@ Expand the block below to view a diagram illustrating the minification process a
 ??? info "Minification Activity Diagram"
     ![minify Task Activity](../images/UI5_Tooling/Task_Minify.svg){ loading=lazy }
 
+
+### Generation of supported locales
+
+The `enhanceManifest` task fills the `supportedLocales` property in the `manifest.json` of a UI5 library/application automatically with the available locales determined by the existence of the respective `.properties` translation file.
+
+#### Scenario: Application
+
+````txt
+- webapp/i18n/
+  - i18n.properties
+  - i18n_en.properties
+  - i18n_en_US.properties
+  - i18n_de.properties
+  - i18n_de_DE.properties 
+````
+
+In the `manifest.json` the `supportedLocales` property will be enhanced as following:
+
+**Source**
+`````json
+"models": {
+  "i18n": {
+    "type": "sap.ui.model.resource.ResourceModel",
+    "settings": {
+	    "bundleName": "sap.ui.demo.todo.i18n.i18n"
+    }
+ }
+`````
+
+**Build Result**
+`````json
+"models": {
+  "i18n": {
+    "type": "sap.ui.model.resource.ResourceModel",
+    "settings": {
+	    "bundleName": "my.app.i18n.i18n"
+	    "supportedLocales": [
+                 "",
+                  "de",
+                  "de_DE",
+                  "en",
+                  "en_US"
+	    ]
+    }
+ }
+`````
+
+To disable the automatic generation of the `supportedLocales`, set `supportedLocales` to any desired value.
```

Comment:
The meaning of an empty array should also be explained here.

Reply:
I would rather add a link to https://ui5.sap.com/#/topic/ec753bc539d748f689e3ac814e129563 instead of explaining runtime code here.